source/javascripts/app/_toc.js

```diff
@@ -29,20 +29,23 @@
     var windowHeight = 0;
     var originalTitle = document.title;
 
-    var recacheHeights = function() {
+    // Dynamically calculate the scroll offset based on the content's top padding
+    scrollOffset = parseInt($(".content").css("padding-top"), 10) + 1;
```

Comment:
Why this change? Are you ignoring the offset given as a parameter to this function?